pkg/provider-local/controller/dummyshoot/add.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dummyshoot
+
+import (
+	"time"
+
+	"github.com/gardener/gardener/extensions/pkg/controller/extension"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+const (
+	// Type is type of the extension.
+	Type string = "dummy-shoot"
```

Comment:
Can we drop "dummy" from this? Rather call it `local-shoot` or `local-ext-shoot`. Also adapt the directory structure (`provider-local/controller/extension/shoot/...`).